submissions/akulahere/html-css-popup/css/main.css

```diff
@@ -0,0 +1,138 @@
+body {
+    font-family: "Times New Roman";
+    margin: 0px;
+}
+
+a {
+    text-decoration: none;
+    color:black;
+}
+
+img {
+    height: 30px;
+    width: 30px;
+}
+
+ul {
+    list-style-type: none;
+    padding-inline-start: 0px;
+}
+.navbar {
+    display: -webkit-flex;
+    display: flex;
+
+    justify-content: flex-end;
+    align-items: center;
+
+    height: 50px;
+    min-height: 50px;
+
+    margin: 0px;
+
+    background: #FFD7A7;
+}
+
+.nav-elem {
+    display: flex;
+    position: relative;
+    align-items: center;
+    padding: 0 30px 0 30px;
+    height: 100%;
+    border: 2px solid transparent;
+    color: black;
+}
+
+.nav-elem:hover {
+    cursor: pointer;
+    opacity: 70%;
+}
+.nav-elem:focus {
+    border-color: darkturquoise;
+}
+.pop-up {
+    position: absolute;
+    top: 50px;
+    left: -100%;
+    width: 280px;
+    max-height: 420px;
+    overflow: auto;
+    z-index: 1;
+    -webkit-transition: 0.5s ease-in-out;
+    -moz-transition: 0.5s ease-in-out;
+    -o-transition: 0.5s ease-in-out;
+    transition: 0.5s ease-in-out;
+    display: none;
+    background-color: lightblue;
+}
+
+.nav-elem:hover .pop-up {
+    display: flex;
```

Comment:
Use flexes for the .pop-up initially, not on the hover .nav-element.

Reply:
.pop-up has display none property, if I change it to display flex it will be shown on the page load.

Reply:
You can add change only display or visibility property on hover, but it's impossible to open pop-up via keyboard
A11y is required in this task

Reply:
Fixed opening pop-up with keyboard or click.